pkg/apis/direct.csi.min.io/v1alpha1/types.go

```diff
@@ -32,37 +32,43 @@ type DirectCSIDrive struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata"`
 
+	OwnerNode      string `json:"ownerNode"`
+	DirectCSIOwned bool   `json:"directcsiOwned"`
+	Path           string `json:"path"`
 	// +optional
 	ModelNumber string `json:"modelNumber,omitempty"`
 	// +optional
 	SerialNumber string `json:"serialNumber,omitempty"`
 	// +optional
-	Name      string `json:"name,omitempty"`
-	OwnerNode string `json:"ownerNode"`
-	// +optional
 	TotalCapacity int64 `json:"totalCapacity,omitempty"`
 	// +optional
 	AllocatedCapacity int64 `json:"allocatedCapacity,omitempty"`
 	// +optional
 	FreeCapacity int64 `json:"freeCapacity,omitempty"`
 	// +optional
-	BlockSize int64  `json:"blockSize,omitempty"`
-	Path      string `json:"path"`
+	BlockSize int64 `json:"blockSize,omitempty"`
 	// +optional
 	RootPartition string `json:"rootPartition,omitempty"`
 	// +optional
 	PartitionNum int `json:"partitionNum,omitempty"`
-	// +optional
-	Filesystem string `json:"filesystem,omitempty"`
-	// +optional
-	Mountpoint string `json:"mountpoint,omitempty"`
 	// +listType=atomic
 	// +optional
 	MountOptions []string `json:"mountOptions,omitempty"`
 	// +optional
 	DriveStatus DriveStatus `json:"driveStatus,omitempty"`
 	// +optional
 	Topology *csi.Topology `json:"topology,omitempty"`
+	// +optional
+	FormatInfo FormatInfo `json:"formatInfo,omitempty"`
```

Comment:
Following our discussion on #direct-csi on slack, let us rename this to RequestedFormat